xenit/solr6/local/93-restore-from-backup.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+set -e
+
+echo "Solr restore from backup start"
+
+if [ -z ${RESTORE_FROM_BACKUP} ]
+then
+    echo "No index backup restore requested, exiting"
+else
+    # if data folder is empty restore from backup
+    if [ -d "${SOLR_DATA_ROOT}/index/alfresco/index" -a `du -s "${SOLR_DATA_ROOT}/index/alfresco/index" | cut -f 1` -gt 100 ]
```

Comment:
There should be more pragmatic ways to check if a folder is empty or not ?

Reply:
The idea was to skip restore if the index folder existed and was bigger than 100 bytes, to restore also in case the index was just created, with a simple segments file + write.lock.
What would be a better way?

Reply:
Is there a case where the index folder has already been filled? Doesn't this script run before solr is started?

Reply:
Yes, it runs before solr has been started. 
The index folder can be filled in case it's a mount or a volume which is reused when restarting the container.